com.unity.cinemachine/Samples~/Shared Assets/Prefabs/Player_Controlled Variant.prefab

```diff
@@ -0,0 +1,82 @@
+%YAML 1.1
```

Comment:
This is a confusing prefab.
It would make more sense to have an "Animated Cameron" prefab, which would look like this:
![image](https://user-images.githubusercontent.com/6424658/212753601-df7db2b0-0989-494f-879e-55fea75eaefd.png)

Then, the Player prefab would be a variant of that, with the extra components added, and the Player tag